recipes/mlconjug3/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "mlconjug3" %}
+{% set version = "3.7.14" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "166e872a2fe5a4fbe035a2afae6b276c97f6290b7e689e864e4baa6416916a27" %}
```

Comment:
Please remove the excess of jinja variables. Anything that is used once should not be a variable. This hinders readability and breaks easily.

BWT try to regenerate this recipe [with grayskull](https://conda-forge.org/blog/posts/2020-03-05-grayskull/), it will make it in the latest recommended format with the minimum amount of tweaking.

Reply:
Thanks @ocefpaf . I will make the required changes.

Reply:
Hi @ocefpaf . I used grayskull to generate the new yaml file.
Thanks again for the feedback.

Cheers,
Sekou.